libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/internal/navigation/NavigationRouteEventListener.kt

```diff
@@ -0,0 +1,19 @@
+package com.mapbox.services.android.navigation.v5.internal.navigation
+
+import okhttp3.Call
+import okhttp3.EventListener
+
+internal class NavigationRouteEventListener @JvmOverloads constructor(
```

Comment:
This https://github.com/mapbox/mapbox-navigation-android/pull/2093#discussion_r331905446 apply here as well, right?

Reply:
I'll mark it with `Fixme` because it still called from `Java` context.